src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -63,8 +65,12 @@ const propTypes = {
 
     /** The transaction from the parent report action */
     transactions: PropTypes.objectOf(transactionPropTypes),
+
     /** List of draft comments */
     draftComments: PropTypes.objectOf(PropTypes.string),
+
+    /** The list of transaction violations */
+    transactionViolations: transactionViolationsPropType,
     ...withCurrentReportIDPropTypes,
```

Comment:
As new lines are added above, let's add here too
```suggestion

    ...withCurrentReportIDPropTypes,
```

Reply:
done